controller/handler/patient.js

```diff
@@ -81,14 +81,17 @@ function patientView (request, reply) {
                 patient: processPatient(currentPatient),
                 trial: currentTrial,
                 surveys: surveyInstances.map((surveyInstance) => {
-                    surveyInstance.startTime = moment(surveyInstance.startTime).format('MM-DD-YYYY');
-                    surveyInstance.endTime = moment(surveyInstance.endTime).format('MM-DD-YYYY');
+                    surveyInstance.startTime = moment(new Date(surveyInstance.startTime)
```

Comment:
moment's default parser is `Date` so wrapping `surveyInstance.startTime` in `Date` seems unnecessary.
this is formatting for the day, so does this need to be `utc`?